sections/staking/components/BurnTiles/BurnTiles.tsx

```diff
@@ -17,6 +25,8 @@ type BurnTilesProps = {
 };
 
 const burnIcon = <Svg src={BurnCircle} />;
+const burnCustomIcon = <Img src={BurnCustomCircle} />; // TODO: investigate why it doesn't render correctly with <Svg /> (ids were replaced to be unique)
```

Comment:
@evgenyboxer could it be the same issue as you told me, the mask-type

Reply:
doesn't seem so... after more testing, it seems that it conflicts with `burn-circle.svg` but I couldn't find any matching ids...